.changeset/famous-hairs-compete.md

```diff
@@ -0,0 +1,7 @@
+---
+'@microsoft/atlas-css': major
```

Comment:
major changes are breaking changes. additional changes are `minor` changes. I think we're looking for `minor` here.

Reply:
changed to minor